api.md

```diff
@@ -840,7 +846,159 @@ SELECT reorder_chunk('_timescaledb_internal._hyper_1_10_chunk', 'conditions_devi
+|`timescaledb.max_interval_per_job`|Max interval per job specifies the amount of data processed by the background materializer job when the continuous aggregate is updated. | Same datatype as the `bucket_width` argument from the `time_bucket` expression.| The default value is `20 * bucket width`.|
+|`timescaledb.create_group_index`|Create indexes on the materialization table for the group by columns (specified by the `GROUP BY` clause of the `SELECT` query). | `BOOLEAN` | Indexes are created by default for every group by expression + time_bucket expression pair.| 
+
+#### Restrictions   
+- `SELECT` query should be of the form specified in the syntax above.
+- The hypertable used in the `SELECT` may not have 
+[row-level-security policies][postgres-rls] enabled. 
+-  `GROUP BY` clause must include a time_bucket expression.  The 
+[`time_bucket`] [time-bucket] expression must use the time dimension column of the hypertable.
+- [`time_bucket_gapfill`][time-bucket-gapfill] is not allowed in continuous
+  aggs, but may be run in a `SELECT` from the continuous aggregate view. 
+- In general, aggregates which can be 
+[parallelized by PostgreSQL][postgres-parallel-agg] are allowed in the view definition, this
+  includes most aggregates distributed with PostgreSQL. Aggregates with `ORDER BY`,
+  `DISTINCT` and `FILTER` clauses are not permitted.  
+* All functions and their arguments included in `SELECT`, `GROUP BY` and `HAVING` clauses must be [immutable][postgres-immutable]. 
+- Queries with `ORDER BY` are disallowed.
+- The view is not allowed to be a [security barrier view][postgres-security-barrier]. 
+[time-bucket]: /api#time_bucket
+[time-bucket-gapfill]: /api#time_bucket_gapfill
+[postgres-immutable]:https://www.postgresql.org/docs/current/xfunc-volatility.html
+[postgres-parallel-agg]:https://www.postgresql.org/docs/current/parallel-plans.html#PARALLEL-AGGREGATION
+[postgres-rls]:https://www.postgresql.org/docs/current/ddl-rowsecurity.html 
+[postgres-security-barrier]:https://www.postgresql.org/docs/current/rules-privileges.html
+
+>:TIP: You can find the [settings for continuous aggregates][timescaledb_information-cagg] and 
+[statistics][timescaledb_information-caggstats] in `timescaledb_information` views.
+
+#### Examples [](cagg-create-examples)
+Create a continuous aggregate view.
+```sql
+CREATE VIEW cagg_view( timec, minl, sumt , sumh )
+WITH ( timescaledb.continuous, timescaledb.refresh_lag = '5 hours', timescaledb.refresh_interval = '1h' )
+AS
+SELECT time_bucket('1day', timec), min(location), sum(temperature),sum(humidity)
+from conditions
+group by time_bucket('1day', timec), location, humidity, temperature;
+```
+---
+## ALTER VIEW (Continuous Aggregate) [](cagg_alter_view)
+`ALTER VIEW` statement can be used to modify the WITH clause  
```

Comment:
These two lines should be one line?


![Screen Shot 2019-05-03 at 11 50 54 AM](https://user-images.githubusercontent.com/12288273/57149440-bb38da80-6d99-11e9-9f76-2e115ec42e2f.png)

Reply:
Also, `WITH` in backticks?

Reply:
changed.